src/Hazelcast.Net.Linq.Async/Visitors/ColumnProjector.cs

```diff
@@ -44,7 +45,7 @@ public ColumnProjector(Func<Expression, bool> canBeColumn)
             if (canBeColumn == null) throw new ArgumentNullException(nameof(canBeColumn));
 
             Nominator = new ExpressionNominator(canBeColumn);
-            _existingAlias = String.Empty;
+            _existingAliases = Enumerable.Empty<string>().ToList();
```

Comment:
`new List<string>()` is probably more efficient?

Reply:
Actually, I just initialize it to suppress compiler warnings and to have some safety. `_existingAliases` is assigned in `Project` method which is the valid way to use class. I've changed it to a `Array.Empty()` according to [benchmark].(https://www.tabsoverspaces.com/233833-best-way-to-create-an-empty-collection-array-and-list-in-csharp-net)